packages/gatsby-theme-api-docs/src/design-system.js

```diff
@@ -0,0 +1,39 @@
+import { designSystem } from '@commercetools-docs/ui-kit';
+
+export const tokens = {
```

Comment:
These tokens should be picked directly from the original uikit tokens, we don't need to re-implement them again.